src/regedit/register.jl

```diff
@@ -417,6 +417,18 @@ function register(
 
         Pkg.Compress.save(compat_file, compat_data)
 
+        reg_pkgs = Pkg.Display.status(Pkg.Types.Context(),
+                                      [PackageSpec("Registrator",
+                                                   Base.UUID("4418983a-e44d-11e8-3aec-9789530b3b3e"))])
+        if length(reg_pkgs) == 0
+            reg_commit = "unknown"
+        else
+            reg_commit = reg_pkgs[1].new.hash
```

Comment:
This is not a commit hash, its a tree hash.

Reply:
How can I get the commit hash?

Reply:
Don't know, I don't think there is an easy way. Why not just use the version instead when `Registrator` is added?